README.md

```diff
@@ -0,0 +1,80 @@
+![Wagtail](https://releases.wagtail.io/wagtail-github-header.png?x)
```

Comment:
Could we have an SVG version of that image?

Reply:
I think all images referenced in the README should be in the repository. It's not much, but I find it better if as many resources as possible are update-able by anyone contributing to the project.

Reply:
There's an SVG version at https://wagtail.io/static/img/wagtail.svg. I'm not sure if I can use this, centred, using Markdown.

I don't mind moving images to the repo but previously @gasman has been strict about reducing repo bloat :)

Reply:
The BrowserStack logo lives in the `.github` folder, I think this would be appropriate for all README images.

Reply:
Here is an example of centering with some HTML:

```html
<h1 align="center">
	<img width="400" src="https://cdn.rawgit.com/sindresorhus/awesome/master/media/logo.svg" alt="Awesome">
	<br>
	<br>
</h1>
```

https://github.com/sindresorhus/awesome/blob/master/readme.md